lib/cucumber/util/colors.js

```diff
@@ -31,7 +31,46 @@ var ConsoleColor = {
     'tag'           : ['cyan']
   },
 
+  hasColor: function () {
```

Comment:
I am going to use https://www.npmjs.com/package/supports-color instead as it seems that has-color has been renamed to this.